### PR TITLE
fix(seed): Add tests, solution, and edit description for Project Euler 16

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Edited the description so the exponents are displayed properly, and
added more tests and a solution.

Here's the solution for QA:

```
function powerDigitSum(exponent) {
  const bigNum = [1];
  let sum = 0;
  
  for (let i = 1; i <= exponent; i++) {
    let count = bigNum.length + 1;
    let overflow = 0;
    for (let j = 0; j < count; j++) {
      let digit = bigNum[j] || 0;
      digit = 2 * digit + overflow;
      
      if (digit > 9) {
        digit -= 10;
        overflow = 1;
      } else {
        overflow = 0;
      }
      
      bigNum[j] = digit;
    }
  }
  
  bigNum.forEach(function(num) {
    return sum += num;
  });
  
  return sum;
}
```

BREAKING CHANGE: None

### CODE REVIEW

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json [hunk 1]
```diff
@@ -597,21 +597,25 @@
...
       "challengeSeed": [
-        "function euler16() {",
+        "function powerDigitSum(exponent) {",
         "  // Good luck!",
         "  return true;",
         "}",
         "",
-        "euler16();"
+        "powerDigitSum(15);",
+        "powerDigitSum(26);",
```
Comment (Bouncey) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: @scissorsneedfoodtoo ideally, we only want to call the challenge function in the seed a maximum of one time

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: @Bouncey, I'm sorry about that. I can go back and change that in this and my other PRs.

**Comment by Bouncey** [comment]
Thank you @scissorsneedfoodtoo 👍

**Review by Bouncey — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
@Bouncey, thank you for taking the time to QA and merge this!

**Pull request merged**